cmake/modules/FindPCRE.cmake

```diff
@@ -61,7 +61,7 @@ if(PCRE_FOUND)
     endif()
     set_target_properties(PCRE::PCRE PROPERTIES
                                      INTERFACE_INCLUDE_DIRECTORIES "${PCRE_INCLUDE_DIR}")
-    if(WIN32)
+    if(WIN32 AND NOT ARCH STREQUAL arm)
```

Comment:
why?

Reply:
Seems it's left from vcpkg era. will chek without this change